examples/tableau/src/model/tableau.ts

```diff
@@ -60,7 +60,10 @@ export class TableauClient {
 
     get token() {
         if (this.loggedIn) {
-            return this.auth.getTokenResponseParam("access_token");
+            const token = this.auth.getTokenResponseParam("access_token");
+            // Refresh token after each use
+            this.refreshToken(); // it's async, don't wait for it to be refreshed...
+            return token;
```

Comment:
Hi Nick, I am curious about a little bit more context here: 😄 
1. does it mean that a new access token is used for new tableau request?
2. what if the new refresh token get expired before being used? Is there any retry here?
Thank you

Reply:
@zhouhao thanks for reviewing
1. Yes, Tableau won't accept reusing the same access token more than once. Weird, I know, but essentially they view that as a replay attack, hence why we refresh the token after every request, so that the next one has a new one.
2. See line 78 in there - if token refresh fails (for whatever reason), the user is logged out and they need to log in again

Reply:
Thank you so much for elaboration @nzapponi 👍